### ISSUE TITLE
Logout from client does not logout from bibdb

### ISSUE THREAD

**Issue by klngwll** [libris/kitin#270]
You stay logged into bibdb, which means you cannot login with another sigel.

**Comment by oBlissing** [comment]
Isn't this pretty standard behaviour for authorization like this?
You can still login as another sigel if you login to bibdb with another sigel.
The issue is probably just a matter of informing the user about how to do it.

Compare to logging into a service via facebook or google. If you log out from the service, you won't be logged out of facebook.

**Comment by niklasl** [comment]
Yes, I agree. We should start by informing the user. But we might also call
logout, or add a link (sign out from bibdb).
